File: frontend/domain/model/Game.js
/** Entidad del dominio, libre de dependencias de interfaz o red. */
export function createGame({
  id,
  name,
  prices = {},
  unavailable_sources = [],
}) {
  return {
    id: String(id),
    name: String(name),
    prices,
    unavailableSources: unavailable_sources,
  };
}

File: backend/app/domain/model/game.py
from typing import Optional


class Game:
    def __init__(
        self,
        id: int,
        name: str,
        prices: dict,
        unavailable_sources: Optional[list[str]] = None,
    ):
        self.id = id
        self.name = name
        self.prices = prices
        self.unavailable_sources = unavailable_sources or []